ncm-metaconfig/src/main/pan/components/metaconfig/schema.pan

```diff
@@ -86,6 +86,18 @@ type ${project.artifactId}_textrender_convert = {
 
 type caf_service_action = string with match(SELF, '^(restart|reload|stop_sleep_start)$');
 
+type ${project.artifactId}_actions = {
+    @{Always run, happens before possible modifications}
```

Comment:
maybe also add ` A failure will cancel any file modification, unless the command is prefixed with -.` here (because it's the case)?

Reply:
Yes, good idea.